### PR TITLE
Update to the Summer16 sample file in RootTools

### PR DESCRIPTION
**Pull request by danbarto**
Added now available samples to the Summer16 python file in RootTools.
Samples which were already present remain unchanged.

### CODE REVIEW

File: RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py [hunk 1]
```diff
@@ -138,16 +138,20 @@
...
 TGJets_ext = kreator.makeMCComponent("TGJets_ext", "/TGJets_TuneCUETP8M1_13TeV_amcatnlo_madspin_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6_ext1-v1/MINIAODSIM", "CMS", ".*root", 2.967)
 tZq_ll_ext = kreator.makeMCComponent("tZq_ll_ext", "/tZq_ll_4f_13TeV-amcatnlo-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6_ext1-v1/MINIAODSIM", "CMS", ".*root", 0.0758 )
 #tZq_nunu = kreator.makeMCComponent("tZq_nunu", "/tZq_nunu_4f_13TeV-amcatnlo-pythia8_TuneCUETP8M1/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM", "CMS", ".*root", 0.1379 )
 tWll = kreator.makeMCComponent("tWll", "/ST_tWll_5f_LO_13TeV-MadGraph-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 0.01123)
+tWnunu = kreator.makeMCComponent("tWnunu", "/ST_tWnunu_5f_LO_13TeV-MadGraph-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 0.01123*1.9822 )
```
Comment (gpetruc) [inline RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py:154]: can you add this below in the `SingleTop` list?

File: RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py [hunk 2]
```diff
@@ -413,7 +424,7 @@
 ## missing xsec QCD_Pt5to10      = kreator.makeMCComponent("QCD_Pt5to10"      , "/QCD_Pt_5to10_TuneCUETP8M1_13TeV_pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM"      , "CMS" , ".*root" , )
 #QCD_Pt10to15     = kreator.makeMCComponent("QCD_Pt10to15"     , "/QCD_Pt_10to15_TuneCUETP8M1_13TeV_pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM"     , "CMS" , ".*root" , 5887580000)
 QCD_Pt15to30          = kreator.makeMCComponent("QCD_Pt15to30"     , "/QCD_Pt_15to30_TuneCUETP8M1_13TeV_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM"     , "CMS" , ".*root" , 1837410000)
-# NOTAVAILYET # QCD_Pt30to50     = kreator.makeMCComponent("QCD_Pt30to50"     , "/QCD_Pt_30to50_TuneCUETP8M1_13TeV_pythia8/RunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v2/MINIAODSIM"      , "CMS" , ".*root" , 140932000)
+QCD_Pt30to50          = kreator.makeMCComponent("QCD_Pt30to50"         , "/QCD_Pt_30to50_TuneCUETP8M1_13TeV_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM"      , "CMS" , ".*root" , 140932000)
```
Comment (gpetruc) [inline RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py:428]: please add this under `QCDPt`

**Review by gpetruc — CHANGES_REQUESTED** [review]

File: RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py (hunk 1 above)
Comment (danbarto) [reply RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py:154]: Done

File: RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py (hunk 2 above)
Comment (danbarto) [reply RootTools/python/samples/samples_13TeV_RunIISummer16MiniAODv2.py:428]: Done

**Comment by danbarto** [comment]
I did the requested changes.
Additionally, I fixed the single top cross sections that were wrong:
- tWch_noFullyHad used the inclusive xsec -> solved
- TBar_tch_powheg used the T_tch_powheg xsec -> solved

**Review by gpetruc — APPROVED** [review]

**Pull request merged**